discussion/app/controllers/CommentsController.scala

```diff
@@ -161,4 +162,11 @@ class CommentsController(csrfConfig: CSRFConfig, val discussionApi: DiscussionAp
       }
     } recover toResult
   }
+
+  // caches "closed" comment threads for an hour.
+  // if the thread is switched on again the url changes and it cache busts itself.
+  private def cacheTime(request: RequestHeader, defaultCacheSeconds: Int) = {
+    val commentsClosed = request.getParameter("closed").contains("true")
+    if (commentsClosed && LongCacheCommentsSwitch.isSwitchedOn) 3800 else defaultCacheSeconds
```

Comment:
ideally keep all integer cache seconds inside Cached.scala with the others, as a val with a helpful name